xc/device.ml

```diff
@@ -1141,6 +1141,11 @@ module PCI = struct
     driver: string;
   }
 
+  type device' =
+    { host:   Xenops_interface.Pci.address
+    ; guest:  int * Xenops_interface.Pci.address option
```

Comment:
How about defining a type name for the `int` to make it clearer what it represents (e.g. `type index = int`)?